test/specs/TickClickingBehaviorSpec.js

```diff
@@ -0,0 +1,101 @@
+describe("TickClickingBehavior", function() {
+	var SLIDER_ID = "testSlider1";
```

Comment:
this spec file is somewhat difficult to read.

Could you break each of these scenarios into their own explicit `it()` statement? I know it might be more repetition but it makes the file much more readable/easier to understand